app/scripts/modules/amazon/src/function/configure/Network.tsx

```diff
@@ -0,0 +1,246 @@
+            subnet.label += ' (deprecated)';
+          }
+        });
+        return subnets.filter(s => s.label);
+      })
+      .then((subnets: ISubnet[]) => {
+        return this.makeSubnetOptions(subnets);
+      });
+
+    const secGroups$ = Promise.resolve(this.getAvailableSecurityGroups());
+    Observable.combineLatest(subnets$, secGroups$)
+      .takeUntil(this.destroy$)
+      .subscribe(([subnets, securityGroups]) => {
+        return this.setState({ subnets, securityGroups });
+      });
+  }
+
+  private handleSubnetUpdate = (options: Array<Option<string>>) => {
+    const subnetsSelected = options.map(o => o.value);
+    this.props.formik.setFieldValue('subnetIds', subnetsSelected);
+  };
+
+  private handleSecurityGroupsUpdate = (options: Array<Option<string>>) => {
+    const sgSelected = options.map(o => o.value);
+    this.props.formik.setFieldValue('securityGroupIds', sgSelected);
+  };
+
+  private handleVpcChange = (vpcId: string): void => {
+    this.props.formik.setFieldValue('vpcId', vpcId);
+    const { subnets } = this.state;
+    const subs = subnets.filter(function(s: ISubnetOption) {
+      return s.vpcId.includes(vpcId);
+    });
+    this.setState({ subnets: subs });
+  };
+
+  private toSubnetOption = (value: ISubnetOption): Option<string> => {
+    return { value: value.subnetId, label: value.subnetId };
+  };
+
+  private getSecurityGroupsByVpc = (sgs: ISecurityGroupsByAccountSourceData): Array<Option<string>> => {
+    const { values } = this.props.formik;
+    const sgOptions: Array<Option<string>> = [];
+    forOwn(sgs, function(sgByAccount, acc) {
+      if (acc === values.credentials) {
+        forOwn(sgByAccount, function(sgByRegion, provider) {
+          if (provider === 'aws') {
+            forOwn(sgByRegion, function(groups, region) {
+              if (region === values.region) {
+                groups.forEach(function(group) {
+                  if (group.vpcId === values.vpcId) {
+                    sgOptions.push({ value: group.id, label: group.name });
+                  }
+                });
+              }
+            });
+          }
+        });
+      }
+    });
+    return sgOptions;
+  };
+
+  public render() {
+    const { vpcOptions, subnets, securityGroups } = this.state;
+    const { values } = this.props.formik;
+    const subnetOptions = (subnets || []).map(this.toSubnetOption);
+    const sgOptions = securityGroups ? this.getSecurityGroupsByVpc(securityGroups) : [];
+    return (
+      <div className="form-group">
+        <div className="col-md-11">
+          <div className="sp-margin-m-bottom">
+            {values.credentials && (
+              <FormikFormField
+                name="vpcId"
+                label="VPC Id"
+                help={<HelpField id="aws.function.vpc.id" />}
+                fastField={false}
+                input={props => (
+                  <ReactSelectInput
+                    inputClassName="cloudfoundry-react-select"
```

Comment:
similar comment: remove `inputClassName` (unless absolutely necessary) and `value` props

Reply:
Done. Will appear in next push.